module1.js

```diff
@@ -54,7 +66,14 @@ function question3(number) {
 
 function question4(array) {
   // TODO:
-}
+  var newArr = [];
+  for (var i = 0; i < array.length; i = i + 2){
+    newArr.push(array[i]);
```

Comment:
Generally speaking _"every second number"_ would start from the _second_ value (`array[1]`), not the first; the test wasn't quite right, though, so good job getting something approximately right.

Reply:
What happened was, I checked the test and tried to match the answer.